tests/backend/test_bosonic_backend.py

```diff
@@ -501,6 +530,34 @@ def test_gkp_wigner_compare_backends(self, fock_eps):
         W_bosonic = gkp_bosonic.wigner(0, xvec, xvec)
         assert np.allclose(W_fock, W_bosonic)
 
+    @pytest.mark.parametrize("fock_eps", [0.1, 0.2, 0.3])
+    @pytest.mark.parametrize("rect_ratio", [0.8, 1, 1.5])
+    def test_rect_gkp_wigner_compare_backends(self, fock_eps, rect_ratio):
+        """Test correct Wigner function are generated by comparing fock and bosonic outputs.
+        Since the fock backend cannot simulate very small epsilon, we restrict to fock_eps>0.1."""
```

Comment:
```suggestion
        """Test that correct Wigner function is generated by comparing fock and bosonic outputs.
        Since the fock backend cannot simulate very small epsilon, we restrict to fock_eps > 0.1."""
```